### DISCUSSION TITLE
[Question]Issues related to release cycle.

### DISCUSSION THREAD

**Discussion by feb-cloud** [flet-dev/flet#3691] [Q&A] [answered]
### Question

Hello, community friends, when I found in the merge record and closed bug problems, many problems have been fixed, but after about two weeks, I did not find that there is a new version of the release, so I would like to ask, does the project have a fixed release plan or cycle? thank you.🤝

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
Atm, there is no release plan, sorry for that.
We will release a pre-release in the coming days. 🤞🏾
